api/token.go

```diff
@@ -517,10 +519,21 @@ func (a *API) issueRefreshToken(ctx context.Context, conn *storage.Connection, u
 	}, nil
 }
 
-func (a *API) setCookieToken(config *conf.Configuration, tokenString string, session bool, w http.ResponseWriter) error {
+// setCookieTokens sets the access_token & refresh_token in the cookies
+func (a *API) setCookieTokens(config *conf.Configuration, token *AccessTokenResponse, session bool, w http.ResponseWriter) error {
+	_ = a.setCookieToken(config, "access_token", token.Token, session, w)
+	_ = a.setCookieToken(config, "refresh_token", token.RefreshToken, session, w)
+	return nil
+}
+
+func (a *API) setCookieToken(config *conf.Configuration, name string, tokenString string, session bool, w http.ResponseWriter) error {
+	cookieName := config.Cookie.Key
+	if name != "" {
```

Comment:
Are we concerned that if `name` is "" then

>    If the user agent receives a new cookie with the same cookie-name,
   domain-value, and path-value as a cookie that it has already stored,
   the existing cookie is evicted and replaced with the new cookie.
   Notice that servers can delete cookies by sending the user agent a
   new cookie with an Expires attribute with a value in the past.

https://datatracker.ietf.org/doc/html/rfc6265#page-8

Which means a refresh token could be set in the access_token. Well, not in the code case, but if elsewhere this function is used.

Perhaps `name` **must** be a required parameter value?

Reply:
good catch, ok will change it to if the name isn't specified, we don't set the cookie